src/vss-pull-requests.service.ts

```diff
@@ -30,7 +30,7 @@ export class VssPullRequests {
         };
         return Promise.all(repos.map(repo => {
             return this.client.getPullRequests(repo.id, search).then(prs => {
-                return prs.map(pr => {
+                return prs.filter(pr => !pr.isDraft).map(pr => {
```

Comment:
Needs to be tested before this PR gets merged.

Reply:
Just tested it on a personal Azure Devops by publishing it privately to my own Publisher.

Reply:
I appreciate that you tested this out! Thank you so much! Makes it much easier.